schemas/context/namespace.example.1.json

```diff
@@ -0,0 +1,4 @@
+{
+  "xdm:id": 12345,
```

Comment:
Why is this just a number and not a URI?

Reply:
Largely for efficiency reasons. User Identities will be processed in the trillions per day and require super low latency.